.github/workflows/tests.yml

```
@@ -11,6 +11,32 @@ on:
       - 'docs/**'
 
 jobs:
+  windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version:
+          - '3.10'
+    name: Windows, SQLite, Python ${{ matrix.python-version }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Get pip cache dir
+        id: pip-cache
+        run: echo "::set-output name=dir::$(pip cache dir)"
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.pip-cache.outputs.dir }}
+          key: ${{ runner.os }}-pip-${{ hashFiles('tests/requirements/py3.txt') }}
```

Support for caching has come on a bit since this was first written...
Some additional options were added to the `actions/setup-python@v2` action.
See https://github.blog/changelog/2021-11-23-github-actions-setup-python-now-supports-dependency-caching/
```suggestion
          cache: pip
          cache-dependency-path: tests/requirements/py3.txt
```

---

Updated.